en/publish/publish-wechatgame-sub-domain.md

```diff
@@ -131,35 +132,35 @@ Since the code and resources of the WeChat open data domain cannot be shared wit
             this.tex.handleLoadedTexture();
             this.display.spriteFrame = new cc.SpriteFrame(this.tex);
         },
-        Update () {
+        update () {
             this._updateSubDomainCanvas();
         }
     });
 ```
 
-## Release Steps
+## Publish Steps
 
-1. Open the main domain project, open the build release panel in `Menu Bar` - `Project`, select the `Wechat Game` platform, and fill in the [Open Data Field Code Directory]. This directory is the path where the open data domain is built, and this path needs to be placed in the main domain build directory. Then click on **Build**.
+1. Open the main context project, open the build release panel in **Menu Bar --> Project**, select the **Wechat Game** platform, and fill in the **Open Data Field Code Directory**. This directory is the path where the open data context is built, and this path needs to be placed in the main context build directory. Then click on **Build**.
 
 ![](./publish-wechatgame/maintest-build.png)
 
-This step will automatically configure the user to the main domain project `build -> wechatgame -> game.json` to identify the directory where the open data domain file is located under the main domain distribution package.
+This step will automatically configure the user to the main context project **build -> wechatgame -> game.json** to identify the directory where the open data context file is located under the main context distribution package.
```

Comment:
This step will help the user automatically configure the **Open Data Context Root** into the main context project **build -> wechatgame -> game.json**......